packages/libs/eda/src/lib/map/analysis/FloatingVizManagement.tsx

```diff
@@ -201,27 +202,36 @@ export default function FloatingVizManagement({
 
   return (
     <>
-      <FloatingDiv
-        style={{
-          top: 200,
-          left: 200,
-        }}
-      >
+      <div>
```

Comment:
**Note to benevolent reviewer(s)**: the changes around here move the "Add a Plot" stuff into the side navigation. More than that, they also add some styling/verbiage tweaks discussed on [2023-03-23's Map UX meeting](https://veupathdb.atlassian.net/wiki/spaces/WG/pages/227573761/2023-03-23+Map+UX).

<img width="941" alt="Screenshot 2023-03-27 at 2 45 14 PM" src="https://user-images.githubusercontent.com/24446573/228049924-02b479ee-5c11-4703-9bc0-cd1e9a2ea4d1.png">